### PR TITLE
Improve table

### PR DESCRIPTION
**Pull request by Liyw979**
**A. introduce a new column in the table**
Before: It's confusing to have 2 identical line here
image
After
image

**B. handle line break**
Before 
image
After
image

### CODE REVIEW

**Comment by fabsx00** [comment]
@max-leuthaeuser I believe you mentioned there is a standard library we can use to shorten `code` strings as well, given that they can become very long when working with `jssrc2cpg`. Can you review this one and see if we can introduce that as part of this PR?

**Comment by max-leuthaeuser** [comment]
Yes, we should use https://commons.apache.org/proper/commons-lang/apidocs/org/apache/commons/lang3/StringUtils.html#normalizeSpace-java.lang.String-
to handle this.
And https://commons.apache.org/proper/commons-lang/apidocs/org/apache/commons/lang3/StringUtils.html#abbreviate-java.lang.String-int- for shortening.

File: dataflowengineoss/src/main/scala/io/joern/dataflowengineoss/language/Path.scala [hunk 1]
```diff
@@ -22,21 +22,22 @@ object Path {
...
         val fileName   = method.file.name.headOption.getOrElse("N/A")
 
-        val trackedSymbol = cfgNode match {
+        val trackedSymbol = (cfgNode match {
           case _: MethodParameterIn =>
             val paramsPretty = method.parameter.toList.sortBy(_.index).map(_.code).mkString(", ")
             s"$methodName($paramsPretty)"
           case _ => cfgNode.statement.repr
-        }
+        }).replace("\n", " ")
```
Comment (max-leuthaeuser) [inline dataflowengineoss/src/main/scala/io/joern/dataflowengineoss/language/Path.scala:38]: See my comment. \n is too simple and will not handle platform dependent line breaks or tabs etc.

**Review by max-leuthaeuser — CHANGES_REQUESTED** [review]

File: dataflowengineoss/src/main/scala/io/joern/dataflowengineoss/language/Path.scala (hunk 1 above)
Comment (Liyw979) [reply dataflowengineoss/src/main/scala/io/joern/dataflowengineoss/language/Path.scala]: Thanks for pointing out this!

**Review by max-leuthaeuser — APPROVED** [review]

**Pull request merged**